src/cosalib/cli.py

```diff
@@ -5,6 +5,50 @@
 import logging as log
 import os
 
+from cosalib.aliyun import (
```

Comment:
Question -- does not need to be changes. Why not:

```python3
from cosalib import aliyun
```

and then use it as:

```python3
print(aliyun.cli)
print(aliyun.run_ore)
print(aliyun.run_ore_replicate)
```

and then later use the module based on the passed in name.

Reply:
Good suggestion...clearly a much better approach. Fixed in the next push.